04-deny-traffic-from-other-namespaces.md

```diff
@@ -20,7 +20,7 @@ pod deployed to.
 Start a web service in namespace default:
 
 ```sh
-$ kubectl run --generator=run-pod/v1 web --namespace default --image=nginx \
+$ kubectl run web --namespace default --image=nginx \
 --labels=app=web --expose --port 80
```

Comment:
Replace --labels=app=web with --labels="app=web"

Reply:
Also --namespace default should become --namespace=default

Reply:
Done